damus/Util/Images/ImageSaver.swift

```diff
@@ -0,0 +1,19 @@
+// ImageSaver.swift
+
+import UIKit
+
+class ImageSaver: NSObject, ObservableObject {
+    @Published var error = false
+
+    func writeToPhotoAlbum(image: UIImage) {
+        UIImageWriteToSavedPhotosAlbum(image, self, #selector(saveCompleted), nil)
+    }
+
+    @objc func saveCompleted(
+        _ image: UIImage,
+        didFinishSavingWithError error: Error?,
+        contextInfo: UnsafeRawPointer
+    ) {
+        self.error = error != nil ? true : false
```

Comment:
There's no reason to put `? true : false` since the expression `error != nil` already evaluates to a `Bool`.
```swift
self.error = error != nil
```